databases_201.rst

```diff
@@ -25,6 +25,84 @@ CouchDB
 Hadoop
 ------
 
+ElasticSearch
+-------------
+
+**Intro**
+
+*ElasticSearch* main logo - "you know, for search". It's near real-time search engine and document-oriented datastore. It's written in Java, so lots of processes depends on JVM (Java Virtual Machine). System has awesome REST API, its documents and queries - all is represented via JSON (https://en.wikipedia.org/wiki/JSON). All configuration can be set up in elasticsearch.yml file. Analogues: Apache Solr, Sphinx.
+
+**Under the microscope**
+
+Elasticsearch is great for scaling. And this is probably one of the most noticable features. You can literally scale elasticsearch cluster for any load. So to begin with, what cluster is?
```

Comment:
Making statements like "great", "probably" and "literally" are a bit unfounded - after all, everything breaks at some point. 😀 Can you re-write the sentence with a some more concrete statements?